src/platform/sdl/main.c

```diff
@@ -321,3 +337,61 @@ static void mSDLDeinit(struct mSDLRenderer* renderer) {
 
 	SDL_Quit();
 }
+
+void setLogger(struct mCore* core) {
+	int fakeBool = 0;
+	bool logToStdout = true;
```

Comment:
This var is unnecessary if we just directly load into _logToStdout.

Reply:
Fixed in https://github.com/mgba-emu/mgba/pull/2345/commits/fac41951f3984e0395765df6c02f0fb02c0ab800 .